Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageVersion Include="Fantomas.Core" Version="[6.0.0]" />
+    <PackageVersion Include="Fantomas.Core" Version="6.1.0-alpha-003" />
```

Comment:
You want to keep the `[` `]` here.
There is no guarantee that any `Fantomas.Core` version is compatible.
[See docs](https://fsprojects.github.io/fantomas/docs/end-users/GeneratingCode.html#Updates)